conn_linux_integration_test.go

```diff
@@ -0,0 +1,67 @@
+//+build integration,linux
+
+package netlink
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+)
+
+func TestLinuxNetlinkMulticast(t *testing.T) {
+	cfg := &Config{
+		Groups: 0x1, // RTMGRP_LINK
+	}
+
+	c, err := Dial(0, cfg) // dials NETLINK_ROUTE
+	if err != nil {
+		t.Fatalf("failed to dial: %v", err)
+
+	}
+
+	in := make(chan []Message)
+
+	// routine for receiving any messages
+	recv := func() {
+		data, er := c.Receive()
+		if er != nil {
+			fmt.Printf("error in receive: %s\n", err)
```

Comment:
`t.Fatalf`, or maybe even `panic(fmt.Sprintf())`.  Not sure what happens when you `t.Fatal` in a goroutine.